src/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  border-radius: 0.5px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars__star:last-of-type {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:first-of-type {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-of-type(-n + 2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-of-type(-n + 3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-of-type(-n + 4) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-of-type(-n + 5) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
[checklist p.2](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

Reply:
Ohh, now I get it))) Sorry...

Reply:
next time - feel free to ask if you're hesitant about the comment